### ISSUE TITLE
`flyctl postgres attach` -> unknown error

### ISSUE THREAD

**Issue by hherman1** [superfly/flyctl#644]
**Describe the bug**
I have a Go http server called `myapp` and a fly postgres db called myappdb. While in the root dir for myapp, I run the following:

```
$ flyctl postgres attach --postgres-app myappdb --verbose
Attaching...⣯ Error An unknown error occured.
```

** Add `fly.toml` **

```toml
app = "myapp"

kill_signal = "SIGINT"
kill_timeout = 5
processes = []

[build]
  builder = "paketobuildpacks/builder:base"
  buildpacks = ["gcr.io/paketo-buildpacks/go:0.12.0"]

[env]
  PORT = "8080"

[experimental]
  allowed_public_ports = []
  auto_rollback = true

[[services]]
  http_checks = []
  internal_port = 8080
  processes = ["app"]
  protocol = "tcp"
  script_checks = []

  [services.concurrency]
    hard_limit = 25
    soft_limit = 20
    type = "connections"

  [[services.ports]]
    handlers = ["http"]
    port = 80

  [[services.ports]]
    handlers = ["tls", "http"]
    port = 443

  [[services.tcp_checks]]
    grace_period = "1s"
    interval = "15s"
    restart_limit = 0
    timeout = "2s"```

**Comment by davissp14** [comment]
@hherman1 I pushed up a rewrite of attach/detach that should be available in flyctl v0.0.263.  Mind giving it a shot?

**Comment by davissp14** [comment]
Please re-open if you run into any additional issues.  It should be noted, however, that if you're running the `postgres-standalone` image, you will need to make sure you're running `v0.0.3' or greater.

You can verify this by running: `fly image show --app <postgres-app>`
